docs/en/sql-reference/statements/delete.md

```diff
@@ -7,57 +7,79 @@ keywords: [delete]
 title: DELETE Statement
 ---
 
+The `DELETE` statement removes rows from the table `[db.]table` that match the expression `expr`.
+
 ``` sql
-DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr
+DELETE FROM [db.]table [ON CLUSTER cluster] WHERE expr;
 ```
 
-`DELETE FROM` removes rows from the table `[db.]table` that match the expression `expr`. The deleted rows are marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background. This feature is only available for the MergeTree table engine family.
+Deleted rows are internally marked as deleted immediately and will be automatically filtered out of all subsequent queries. Cleanup of data happens asynchronously in the background until the next merge. As a result, it is possible that for an unspecified period, data is not actually deleted from storage and is only marked as deleted.
+
+The `DELETE` statement is only available for the *MergeTree table engine family.
 
-For example, the following query deletes all rows from the `hits` table where the `Title` column contains the text `hello`:
+**Do not use the DELETE statement frequently as it can negatively affect ClickHouse performance.**
+
+## Examples
 
 ```sql
+-- Deletes all rows from the `hits` table where the `Title` column contains the text `hello`
 DELETE FROM hits WHERE Title LIKE '%hello%';
 ```
 
-Lightweight deletes are asynchronous by default. Set `mutations_sync` equal to 1 to wait for one replica to process the statement, and set `mutations_sync` to 2 to wait for all replicas.
```

Comment:
Why were all mentions of `DELETE` being lightweight deleted?

Reply:
Sorry, I'm not sure I understand this comment. Can you expand further?

Reply:
I mean previously DELETEs in the text were explicitly called lightweight, now it is just DELETEs (except line 75)